src/mscorlib/Common/PinnableBufferCache.cs

```diff
@@ -21,22 +21,6 @@ namespace System
 {
     internal sealed class PinnableBufferCache
     {
-        /// <summary>
-        /// Create a new cache for pinned byte[] buffers
-        /// </summary>
-        /// <param name="cacheName">A name used in diagnostic messages</param>
-        /// <param name="numberOfElements">The size of byte[] buffers in the cache (they are all the same size)</param>
-        public PinnableBufferCache(string cacheName, int numberOfElements) : this(cacheName, () => new byte[numberOfElements]) { }
```

Comment:
Yes, you're right.  I forgot that it is used by Overlapped.  In that case, do you mind keeping the public methods and ifdef them out (or just keep them because they're small and the class is internal)?  There are a handful of copies of this class around and it would be good to have these around in case we need to use them.  That way we don't have to copy them from another source.